compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -19318,8 +19407,16 @@ OMR::Z::TreeEvaluator::vsetelemEvaluator(TR::Node *node, TR::CodeGenerator *cg)
                }
             else if (size == 4)
                {
-               uint32_t value = valueNode->getIntegerNodeValue<uint32_t>();
-               offset = cg->fe()->findOrCreateLiteral(cg->comp(), &value, size);
+               if (valueNode->getOpCode().isFloat())
+                  {
+                  float value = valueNode->getFloat();
```

Comment:
I used value in vsetelem because the other cases in vsetelem used value. Similarly the existing cases in getvelem used 'v' so I used 'v' in getvelem. I will change the 'v' in getvelem to "value" to match vsetelem and because it's more descriptive than just 'v'.